com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -49,6 +50,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Fixed
 
+- Fixed issue where maxCapacity calculation overflows if a developer sets a very, very high (large) m_DisconnectTimeoutMS in the Editor for Unity Transport. (#3810)
```

Comment:
This changelog entry has a bit too much technical detail. We're more interested in explaining what's changed for users 😄 

```suggestion
- Fixed an integer overflow that occurred when configuring a large disconnect timeout for Unity Transport. (#3810)
```